apps/sms/views/conversation/js/compose.js

```diff
@@ -912,11 +1045,16 @@ var Compose = (function() {
     }
   });
 
+  Object.defineProperty(compose, 'previousSegment', {
```

Comment:
Why do we need to expose this as public property, do you think it's going to be used outside of Compose? + I think it's a chance to revisit other Compose's publicly available properties, I think we don't need some others as well.